specs/notation-cli.md

```diff
@@ -34,7 +34,7 @@ Available Commands:
   login       Login to registry
   logout      Log out from the logged in registries
   plugin      Manage plugins
-  policy      [Preview] Manage trust policy configuration for signature verification
+  policy      [Experimental] Manage trust policy configuration for signature verification
```

Comment:
How about mark `Experimental` for `notation policy` subcommands, like `show` and `import`?